src/pages/store/index.vue

```diff
@@ -31,50 +31,7 @@
 
           <!-- Desktop Filter & Sorting -->
           <div class="hidden desktop:flex items-center gap-[16px] relative">
-            <!-- Search button -->
-            <ButtonV2 preset="tertiary" @click="toggleSearch">
-              <IconSearch />
-            </ButtonV2>
-            <!-- Search bar -->
-            <div
-              :style="{
-                width: isSearchOpen ? '100%' : '0',
-              }"
-              :class="[
-                'absolute',
-                'top-0',
-                'left-0',
-                'z-10',
-                'flex',
-                'items-center',
-                'bg-shade-gray',
-                'gap-[12px]',
-                isSearchOpen && 'pl-[16px]',
-                'h-full',
-                'rounded-[10px]',
-                'transition-[width]',
-                'duration-[5000]',
-                'ease-in-out',
-              ]"
-            >
-              <IconSearch />
-              <input
-                :value="searchQuery"
-                class="w-full bg-transparent border-0 focus-visible:outline-none"
-                type="text"
-                :placeholder="$t('gutenberg_search_placeholder')"
-                @input="debouncedUpdateSearchKeyword"
-              />
-              <ButtonV2
-                v-if="isSearchOpen"
-                size="tiny"
-                preset="plain"
-                @click="toggleSearch"
-              >
-                <IconClose class="cursor-auto" />
-              </ButtonV2>
-            </div>
-
+            <Search />
```

Comment:
Prefer 2 words for components, [ref](https://eslint.vuejs.org/rules/multi-word-component-names)
```suggestion
            <SearchBar />
```